packages/vx-annotation/test/LinePathAnnotation.test.tsx

```diff
@@ -65,4 +65,15 @@ describe('<LinePathAnnotation />', () => {
       ),
     ).toBe(true);
   });
+
+  test('it should have default x and y accessors', () => {
```

Comment:
We could use `it(...)` function instead of `test(...)` too so it will read more like

```ts
it('should have default x and y accessors', ...)
```

Reply:
sure, i don't have a preference